### DISCUSSION TITLE
Serbian translation

### DISCUSSION THREAD

**Discussion by medicmilos** [vuejs-translations/guidelines#27] [Ideas]
Hey all, is anyone interested to make Serbian translation? :) 

Feel free to join Serbian VueJS community [vuejs-rs](https://github.com/vuejs-rs)

**Comment by StokicDusan** [comment]
Very interesting! :thinking:

**Comment by srdjanmaravic** [comment]
Interested, you can count me in if you need assistance.

**Comment by rammba** [comment]
Hello @medicmilos, is this still active? I can't access [vuejs-rs](https://github.com/vuejs-rs) organization. Is it private or removed maybe?

**Comment by TMinarik00** [comment]
I would love to help if anybody is still doing this.
